stm/stm.ml

```diff
@@ -2577,7 +2577,8 @@ type stm_init_options = {
 
   (* Initial load path in scope for the document. Usually extracted
      from -R options / _CoqProject *)
-  iload_path   : Loadpath.coq_path list;
+  ml_load_path   : CUnix.physical_path list;
+  vo_load_path   : Loadpath.vo_path list;
```

Comment:
Is the comment still up to date? I guess it should be changed to `Inital load paths` plural.

Reply:
I've updated the comments and ported them to ocamldoc.